app/models/notification.rb

```diff
@@ -0,0 +1,6 @@
+class Notification < ApplicationRecord
+  belongs_to :sender, class_name: 'User'
```

Comment:
@FaxriddinMaxmadiyorov before push please run rubocop and rspec

Reply:
@XaoGao ok